phpcs.xml

```diff
@@ -5,9 +5,11 @@
     <file>.</file>
 
     <!-- Ignore these directories -->
-    <exclude-pattern>dist/*</exclude-pattern>
+    <exclude-pattern>*/dist/*</exclude-pattern>
     <exclude-pattern>node_modules/*</exclude-pattern>
-    <exclude-pattern>vendor/*</exclude-pattern>
+    <exclude-pattern>*/tests/*</exclude-pattern>
+    <exclude-pattern>*/vendor/*</exclude-pattern>
```

Comment:
Having both a `vendor` and ` vendors` directory is going to get confusing...

Reply:
@GaryJones We have folders in the codebase that are named `vendor`.  We'll correct the naming to standardize in v2.